modules/social_features/social_group/social_group.install

```diff
@@ -811,3 +811,12 @@ function social_group_update_10308() {
   // Set display mode for group content to display on stream.
   activity_creator_set_entity_view_mode('group_content', 'activity');
 }
+
+/**
+ * Disable the "group_nodes" view.
+ */
+function social_group_update_10309() {
```

Comment:
@rollki I see we already have similar code in `social_group_update_8002()` and in the `social_group_install()`, but it looks like it doesn't always work.
Maybe it will be more stable to unset this tab in the `social_group_menu_local_tasks_alter()`?

Reply:
@ribel 
Yep, you are right, let's do it!